### PR TITLE
schema support Chinese

### PR DESCRIPTION
**Pull request by cooper-lzy**

### CODE REVIEW

File: docs-2.0/3.ngql-guide/9.space-statements/1.create-space.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ CREATE SPACE [IF NOT EXISTS] <graph_space_name> (
 |Parameter|Description|
 |:---|:---|
 |`IF NOT EXISTS`|Detects if the related graph space exists. If it does not exist, a new one will be created. The graph space existence detection here only compares the graph space name (excluding properties).|
-|`<graph_space_name>`|Uniquely identifies a graph space in a Nebula Graph instance. The name of the graph space is case-sensitive and allows letters, numbers, or underlines. [Keywords and reserved words](../../3.ngql-guide/1.nGQL-overview/keywords-and-reserved-words.md) are not allowed.|
+|`<graph_space_name>`|Uniquely identifies a graph space in a Nebula Graph instance. The name of the graph space is case-sensitive and allows Chinese, letters, numbers, or underlines. When including Chinese or using reserved keywords alone, quote them with backticks. For more informartion, see [Keywords and reserved words](../../3.ngql-guide/1.nGQL-overview/keywords-and-reserved-words.md).|
```
Comment (abby-cyber) [inline docs-2.0/3.ngql-guide/9.space-statements/1.create-space.md:26]:
```suggestion
|`<graph_space_name>`|Uniquely identifies a graph space in a Nebula Graph instance. The name of the graph space is case-sensitive and allows Chinese, letters, numbers, or underlines. When using Chinese or reserved keywords, quote them with backticks. For more information, see [Keywords and reserved words](../../3.ngql-guide/1.nGQL-overview/keywords-and-reserved-words.md).|
```

File: docs-2.0/3.ngql-guide/9.space-statements/1.create-space.md [hunk 2]
```diff
@@ -58,7 +58,7 @@ CREATE SPACE <new_graph_space_name> AS <old_graph_space_name>;
 
 | Parameter | Description |
 | :--- | :--- |
-| `<new_graph_space_name>` | The name of the graph space that is newly created. The name of the graph space is case-sensitive and allows letters, numbers, or underlines. [Keywords and reserved words](../../3.ngql-guide/1.nGQL-overview/keywords-and-reserved-words.md) are not allowed. When a new graph space is created, the schema of the old graph space `<old_graph_space_name>` will be cloned, including its parameters (the number of partitions and replicas, etc.), Tag, Edge type, and native indexes. |
+| `<new_graph_space_name>` | The name of the graph space that is newly created. The name of the graph space is case-sensitive and allows Chinese, letters, numbers, or underlines. When including Chinese or using reserved keywords alone, quote them with backticks. For more informartion, see [Keywords and reserved words](../../3.ngql-guide/1.nGQL-overview/keywords-and-reserved-words.md). When a new graph space is created, the schema of the old graph space `<old_graph_space_name>` will be cloned, including its parameters (the number of partitions and replicas, etc.), Tag, Edge type, and native indexes. |
```
Comment (abby-cyber) [inline docs-2.0/3.ngql-guide/9.space-statements/1.create-space.md:61]:
```suggestion
| `<new_graph_space_name>` | The name of the graph space that is newly created. The name of the graph space is case-sensitive and allows Chinese, letters, numbers, or underlines. When using Chinese or reserved keywords, quote them with backticks. For more information, see [Keywords and reserved words](../../3.ngql-guide/1.nGQL-overview/keywords-and-reserved-words.md). When a new graph space is created, the schema of the old graph space `<old_graph_space_name>` will be cloned, including its parameters (the number of partitions and replicas, etc.), Tag, Edge type, and native indexes. |
```

**Review by abby-cyber — APPROVED** [review]

File: docs-2.0/3.ngql-guide/1.nGQL-overview/keywords-and-reserved-words.md [hunk 3]
```diff
@@ -6,7 +6,9 @@ Non-reserved keywords are permitted as identifiers without quoting. To use reser
 
 !!! Note
 
-    Keywords are case-insensitive.
+    - Keywords are case-insensitive.
+
+    - To use Chinese characters, it's required to quote them with backticks.
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/1.nGQL-overview/keywords-and-reserved-words.md:11]:
```suggestion
    - To use Chinese characters, put them in backticks.
```

Comment (whitewum) [reply docs-2.0/3.ngql-guide/1.nGQL-overview/keywords-and-reserved-words.md:11]: Non-English (e.g, Chinese)

File: docs-2.0/3.ngql-guide/1.nGQL-overview/keywords-and-reserved-words.md [hunk 4]
```diff
@@ -8,7 +8,7 @@ Non-reserved keywords are permitted as identifiers without quoting. To use reser
 
     - Keywords are case-insensitive.
 
-    - To use Chinese characters, put them in backticks.
+    - To use Non-English (e.g, Chinese), put them in backticks.
```
Comment (randomJoe211) [inline docs-2.0/3.ngql-guide/1.nGQL-overview/keywords-and-reserved-words.md:11]:
```suggestion
    - To use Non-English (e.g, Chinese) characters, put them in backticks.
```

**Review by abby-cyber — APPROVED** [review]

**Review by abby-cyber — APPROVED** [review]

**Review by izhuxiaoqing — APPROVED** [review]

**Pull request merged**